projects/vdk-core/tests/functional/run/test_run_sql_queries.py

```diff
@@ -75,6 +75,8 @@ def db_connection_recover_operation(self, recovery_cursor: RecoveryCursor) -> No
 
```

Comment:
take a look at the `CONTRIBUTING.md` to find the git commit template format,

Reply:
Done